### ISSUE TITLE
[BUG] Injection Warning as opcode has incompatible changes

### ISSUE THREAD

**Issue by MrLoLf** [Traben-0/Entity_Texture_Features#141]
I am using the latest version, can be checked in the log.

It shows in the log a 
```
[Render thread/WARN]: Injection warning: LVT in net/minecraft/class_922::method_24302(Lnet/minecraft/class_1309;ZZZ)Lnet/minecraft/class_1921; has incompatible changes at opcode 122 in callback entity_texture_features-common.mixins.json:entity.renderer.MixinLivingEntityRenderer from mod entity_texture_features->@Inject::etf$renderLayerAlter(Lnet/minecraft/class_1309;ZZZLorg/spongepowered/asm/mixin/injection/callback/CallbackInfoReturnable;Lnet/minecraft/class_2960;)V.
 
Expected: [Lnet/minecraft/class_2960;]
    Found: []
Available: []
```

- What happened? This WARN shows up.
- What were you doing when it happened? Starting my game.

**Crash log**
[latest.log](https://github.com/Traben-0/Entity_Texture_Features/files/10517321/latest.log)
With only the mod [latest.log](https://github.com/Traben-0/Entity_Texture_Features/files/10517343/latest.log)

**Your setup:**

- Minecraft version: 1.19.3
- ModLoader: fabric
- ETF Version 4.3.1
- Does the issue persist with only ETF installed? Yes, as can be seen here [latest.log](https://github.com/Traben-0/Entity_Texture_Features/files/10517343/latest.log)
- Have you checked the FAQ in the Readme, or Discord, to see if it is a known issue? Yes

**Comment by Traben-0** [comment]
I get this too for some reason the mixin still works for me just fine, is it breaking something for you?

I will try and narrow down the cause for the warning

**Comment by MrLoLf** [comment]
No it's working fine, but I thought I post this here as no one has done it yet, so you would be definitely aware of it. And it could probably get fixed.

**Comment by Traben-0** [comment]
Fixed in 4.4
